### PR TITLE
French translation fixed

### PR DESCRIPTION
**Pull request by Anoukmch**

### CODE REVIEW

File: frontend/src/views/TournamentsView.vue [hunk 1]
```diff
@@ -117,6 +120,7 @@ onMounted(() => {
 	});
 	fetchData();
 })
+>>>>>>> 1aa407da95a6d8d6f90473279b656866eef1b42e
```
Comment (PythonGermany) [inline frontend/src/views/TournamentsView.vue:123]: Some sort of leftover from a merge.
```suggestion
```

File: frontend/src/views/TournamentsView.vue [hunk 2]
```diff
@@ -87,6 +87,9 @@ const resetInputFields = () => {
    input.value.description = '';
 };
 
+<<<<<<< HEAD
+onMounted(fetchData)
+=======
```
Comment (PythonGermany) [inline frontend/src/views/TournamentsView.vue:92]: Some sort of leftover from a merge.
```suggestion
```

File: elastic-stack/.env
Comment (PythonGermany) [inline elastic-stack/.env:1]: This file should not be here.

File: backend/transc/api/models.py [hunk 3]
```diff
@@ -16,6 +16,7 @@ def get_avatar_path(instance, filename):
 
 	username = models.CharField(max_length=12, unique=True, null=False)
 	nickname = models.CharField(max_length=12, unique=True, null=True)
+	nickname = models.CharField(max_length=11, unique=True, null=True)
```
Comment (PythonGermany) [inline backend/transc/api/models.py:19]: Duplicate nickname field

File: frontend/src/views/SettingsView.vue [hunk 4]
```diff
@@ -118,6 +118,14 @@ const changeAvatar = async(event) => {
...
 							</div>
 						</div>
+						<div class="col-6 col-sm-12">
+							<div class="form-group row mt-3 align-items-center">
+								<label class="col-md-4 col-sm-12 control-label">Number: </label>
+								<div class="col-md-8  col-sm-12">
+									<input type="text" class="form-control" :placeholder="'Your number'" v-model="input.nickname">
+								</div>
+							</div>
+						</div>
```
Comment (PythonGermany) [inline frontend/src/views/SettingsView.vue:128]: What is the purpose of this?

**Review by jmatheis00 — APPROVED** [review]

**Review by jmatheis00 — APPROVED** [review]

**Pull request merged**